### PR TITLE
e2e: Support increasing number of parallel tests to improve run times

### PR DESCRIPTION
**Pull request by bsessions85**
#### Changes proposed in this Pull Request
* There are some changes to packages and run.sh that will allow for splitting tests across containers (for use in wp-e2e-tests-for-branches)

#### Testing instructions
* Try using `./run.sh -g -f spec/wp-signup-spec.js,spec/wp-reader-spec.js' locally and make sure that only tests from those two files run.

### CODE REVIEW

File: test/e2e/package.json [hunk 1]
```diff
@@ -60,7 +60,7 @@
 		"spec-xunit-reporter": "./lib/reporter",
 		"testarmada-magellan": "11.0.1",
 		"testarmada-magellan-local-executor": "^2.0.0",
-		"testarmada-magellan-mocha-plugin": "github:Automattic/magellan-mocha-plugin#660cf2c",
+		"testarmada-magellan-mocha-plugin": "github:Automattic/magellan-mocha-plugin#fa17dd9",
```
Comment (Stojdza) [inline test/e2e/package.json]: `fa17dd9` is related to specific commit in Automattic/magellan-mocha-plugin repo, right? Should this be removed when it goes to master branch or to stay like this?

**Review by Stojdza — APPROVED** [review]
Left one comment. This looks good and works locally as expected 👍

**Comment by blowery** [comment]
Is the description and title still accurate for this PR? I don't see any alterations to the thread count in the current diff...

**Comment by bsessions85** [comment]
Thanks @blowery, The threads piece went in another PR so this is smaller. Most of the actual work to split is done in the e2e bridge repo.

**Pull request merged**